libs/core/src/lib/avatar-group/components/avatar-group-host.component.ts

```diff
@@ -0,0 +1,151 @@
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ContentChildren,
+    DestroyRef,
+    ElementRef,
+    inject,
+    Input,
+    OnChanges,
+    QueryList,
+    signal,
+    SimpleChanges,
+    ViewEncapsulation
+} from '@angular/core';
+import { NgIf } from '@angular/common';
+import { animationFrames, combineLatest, delayWhen, map, Observable, startWith, Subject } from 'rxjs';
+import { HasElementRef, ResizeObserverDirective } from '@fundamental-ngx/cdk/utils';
+import { AvatarGroupItemDirective } from '../directives/avatar-group-item.directive';
+import { AvatarGroupItemRendererDirective } from '../directives/avatar-group-item-renderer.directive';
+import { AvatarGroupHostConfig } from '../types';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+
+@Component({
+    selector: 'fd-avatar-group-host',
+    host: {
+        class: 'fd-avatar-group',
+        '[class.fd-avatar-group--individual-type]': 'type === "individual"',
+        '[class.fd-avatar-group--group-type]': 'type === "group"',
+        '[class.fd-avatar-group--horizontal]': 'orientation === "horizontal"',
+        '[class.fd-avatar-group--vertical]': 'orientation === "vertical"',
+        '[class.fd-avatar-group--xs]': 'size === "xs"',
+        '[class.fd-avatar-group--s]': 'size === "s"',
+        '[class.fd-avatar-group--m]': 'size === "m"',
+        '[class.fd-avatar-group--l]': 'size === "l"',
+        '[class.fd-avatar-group--xl]': 'size === "xl"'
+    },
+    template: '<ng-content></ng-content>',
+    standalone: true,
+    imports: [NgIf],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    hostDirectives: [ResizeObserverDirective],
+    encapsulation: ViewEncapsulation.None
+})
+export class AvatarGroupHostComponent implements AfterViewInit, OnChanges, HasElementRef, AvatarGroupHostConfig {
+    /** @hidden */
+    @Input()
+    type: AvatarGroupHostConfig['type'];
+
+    /** @hidden */
+    @Input()
+    orientation: AvatarGroupHostConfig['orientation'];
+
+    /** @hidden */
+    @Input()
+    size: AvatarGroupHostConfig['size'];
+
+    /** @hidden */
+    @Input()
+    resizeEmitter: Observable<ResizeObserverEntry[]> = inject(ResizeObserverDirective).resized;
+
+    /** @hidden */
+    @Input()
+    items: QueryList<AvatarGroupItemDirective>;
+
+    /** @hidden */
+    @ContentChildren(AvatarGroupItemRendererDirective, { descendants: true })
+    portals: QueryList<AvatarGroupItemRendererDirective>;
+
+    /** @hidden */
+    elementRef = inject(ElementRef);
```

Comment:
for all `hidden` items we use `_` prefix

Reply:
This comes from the interface, better would be to add repetitive jsdoc just to align with the commonness